### DISCUSSION TITLE
Permalink of a draft release

### DISCUSSION THREAD

**Discussion by achimnol** [community/community#5466] [Other Feature Feedback, Questions, & Ideas] [answered]
I'd like to share the draft release note in GitHub with my organization members.
The problem is that I cannot do it, because whenever I update the **description** (not only the title!) the URL's hash part is changed (e.g., `c587...` in `https://github.com/<org>/<repo>/releases/tag/untagged-c587caf1286dd83d3cd4`).
So I often just share the link to the all releases page, but this feels not right.

Also I'd like to be able to share a draft release note _publicly_, not only to organization members, [as in this discussion](https://github.com/github/feedback/discussions/4192).

**Comment by MylesBorins** [comment, marked as answer]
Hey @achimnol sorry for not responded sooner, I thought i had but totally missed it.

At this time we are not looking into making changes to drafts as we have other work items that are higher priority (such as improving search + version detection for auto-generated release notes).

I'm going to mark this as answered as we won't be doing work related to drafts today, but I'll come back and let you know if that changes.
